automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -214,7 +214,7 @@ def product_upgrade(product):
                                 upgraded = str(execute(
                                     get_sat_cap_version, 'cap', host=cap_host
                                     )[cap_host])
-                                if current:
+                                if current != 'None':
```

Comment:
AFAIK, we should not do this type cast `None` to `'None'` string and then check for if `current != 'None'`. This sounds like something is not right.
I would suggest we typecast it here , https://github.com/SatelliteQE/automation-tools/blob/master/automation_tools/satellite6/upgrade/tasks.py#L721 
and 
also let that function return a `None` value if version is `NULL` or `''` or not found or 'Unavailable' for some reason. For example if `satellite` package itself is missing for some reason.

```
if version != 'Unavailable':
    return str(version)
```
Can you please check if we can do this?  
So with this change if `version == ''` or `version == 'Unavailable'` , `get_sat_cap_version` will return `None`. So that we should be able to do just `if current:`.